code/API_definitions/qod-api.yaml

```
@@ -206,6 +206,12 @@ paths:
                     status: 400
                     code: OUT_OF_RANGE
                     message: "Invalid port ranges specified: devicePorts"
+                DurationOutOfRangeForQoSProfile:
+                  summary: The requested duration is out of the allowed range for the specific QoS profile 
```

```suggestion
                  summary: The requested duration is out of the allowed range for the specific QoS profile
```
To make the linter happy ...